doc/dftb+/manual/dftbp.tex

```diff
@@ -2539,10 +2543,10 @@ \section{Options}
   value can be used to reproduce earlier MD calculations by setting
   the initial seed to the same value.)
 
-\item[\is{MinimiseMemoryUsage}] Tries to minimise memory usage by
-  storing various matrices on disc instead of keeping them in memory.
-  Set it to \is{Yes} to reduce the memory requirement for calculations
-  with many k-points or spin polarisation.
+\item[\is{TimingVerbosity}] Level of information regarding timings of
+  sections of the code, higher values becoming more verbose. Setting
+  this parameter to be below 0 supresses any information being
```

Comment:
At 0 or below

Reply:
Changed. I also changed the default timing verbosity to 0 in production code.